### ISSUE TITLE
Problem with Python 3 + 2.49 + Chromedriver on Ubuntu 14.04

### ISSUE THREAD

**Issue by cjauvin** [SeleniumHQ/selenium#1476]
When I try to execute this script with Python 3:

```
from xvfbwrapper import Xvfb
from selenium import webdriver

display = Xvfb(nolisten='tcp')
display.start()
driver = webdriver.Chrome()
driver.get('http://www.google.com')
driver.quit()
display.stop()
```

I get this error:

```
$ python3 toto.py
Exception ignored in: <bound method Service.__del__ of <selenium.webdriver.chrome.service.Service object at 0x7f63613610f0>>
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/dist-packages/selenium/webdriver/common/service.py", line 136, in __del__
  File "/usr/local/lib/python3.4/dist-packages/selenium/webdriver/common/service.py", line 120, in stop
  File "/usr/local/lib/python3.4/dist-packages/selenium/webdriver/common/service.py", line 95, in send_remote_shutdown_command
  File "<frozen importlib._bootstrap>", line 2237, in _find_and_load
  File "<frozen importlib._bootstrap>", line 2222, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 2155, in _find_spec
TypeError: 'NoneType' object is not iterable
```

Here are some informations about my environment:

```
$ pip3 show selenium

---
Metadata-Version: 1.1
Name: selenium
Version: 2.49.0
Summary: Python bindings for Selenium
Home-page: https://github.com/SeleniumHQ/selenium/
Author: UNKNOWN
Author-email: UNKNOWN
License: UNKNOWN
Location: /usr/local/lib/python3.4/dist-packages
Requires:

$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 14.04.3 LTS
Release:        14.04
Codename:       trusty

$ python3 --version
Python 3.4.3

$ chromedriver --version
ChromeDriver 2.20.353124 (035346203162d32c80f1dce587c8154a1efa0c3b)

$ chromium-browser --version
Chromium 47.0.2526.106 Ubuntu 14.04
```

Any idea what's causing this? It works well with Python 2.

**Comment by muttonchop** [comment]
I am also seeing this exact issue (reproducible with code provided) on OSX 10.10.5

**Comment by xc8tlik** [comment]
+1 Selenium 2.53.2, Oracle Linux (RHEL) 7.2, ChromeDriver 2.21.371461. Fixed by downgrade to 2.48.

**Comment by lmtierney** [comment]
I'm guessing this is resolved as it is old and there are many new selenium & chromedriver versions. Please open a new issue if you are still seeing this with the latest versions.
